### DISCUSSION TITLE
Does crazyswarm simulate sensor values like gyroscope, accelerator and controller part?

### DISCUSSION THREAD

**Discussion by ElaineYao** [USC-ACTLab/crazyswarm#630] [Q&A] [answered]
Hi,

I was wondering if the simulation version of crazyswarm simulates the following parts.
1. Sensor readings like gyroscope, accelerator, and position by motion-capture.
2. Attitude and Position controller part. 

If so, I was wondering if you could point me to the corresponding folder/files. Thanks,

**Comment by whoenig** [comment, marked as answer]
No, there is currently no physics simulation. You can take a look at CrazyS for such capabilities.
